benefits/core/migrations/0002_sample_data.py

```diff
@@ -147,8 +147,6 @@ def load_sample_data(app, *args, **kwargs):
         unverified_title=_("eligibility.pages.unverified.mst_cc.title"),
         unverified_headline=_("eligibility.pages.unverified.mst_cc.headline"),
         unverified_blurb=_("eligibility.pages.unverified.mst_cc.p[0]"),
-        eligibility_confirmed_item_heading=_("enrollment.pages.index.mst_cc.eligibility_confirmed_item.heading"),
-        eligibility_confirmed_item_details=_("enrollment.pages.index.mst_cc.eligibility_confirmed_item.details"),
```

Comment:
I don't understand how this data can be deleted here in the data migration without making the fields `null=True` in the model/schema migration... but it does work 🤔

Reply:
Yeah, I'm not sure either...

Reply:
Can you try to find out? Doesn't have to be today / part of this PR.

Reply:
So the short answer is: somehow the columns have an empty-string value.

```
sqlite> select eligibility_confirmed_item_heading from core_eligibilityverifier;
eligibility_confirmed_item_heading                                 
-------------------------------------------------------------------
enrollment.pages.index.mst_login.eligibility_confirmed_item.heading
                                                                   
sqlite> select count(eligibility_confirmed_item_heading) from core_eligibilityverifier where eligibility_confirmed_item_heading="";
count(eligibility_confirmed_item_heading)
-----------------------------------------
1                                        
sqlite> select count(eligibility_confirmed_item_heading) from core_eligibilityverifier where eligibility_confirmed_item_heading<>"";
count(eligibility_confirmed_item_heading)
-----------------------------------------
1                                        
sqlite> select count(eligibility_confirmed_item_heading) from core_eligibilityverifier where eligibility_confirmed_item_heading is null;
count(eligibility_confirmed_item_heading)
-----------------------------------------
0                   
```
I think I'll just put `null=False` on the fields so that it's not confusing.